scripts/setup/macos/install_viewer.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+set -eu
+
+# Install cbmc-viewer
+
+# Source kani-dependencies to get CBMC_VIEWER_VERSION
+source kani-dependencies
```

Comment:
If we're not installing a specific version, then we shouldn't need to source kani-dependencies.